tests/assets/move-projects/signer-scripts/sources/Generics.move

```diff
@@ -0,0 +1,66 @@
+script {
+    use std::signer;
+
+    // Should fail!
+    fun signer_before_generic<T: copy + drop>(s: signer, x: T) {
+        let _y = x;
+
+        // A way to make some use of this signer.
+        assert!(signer::address_of(&s) == @0xCAFE, 0);
+    }
+}
+
+script {
+    use std::signer;
+
+    // Should fail!
+    fun two_signers_before_generic<T: copy + drop>(s1: signer, s2: signer, x: T) {
```

Comment:
Is there any point in adding a test with two singers before generic if we've tested one signer before generic?
As there can always be question why there are one and two signers tested but why not three, four etc. 😸

Reply:
True - Let's remove it!
Done